app/assets/stylesheets/custom.scss

```diff
@@ -7,3 +7,7 @@ body {
   padding: 2.5px 15px;
   position: absolute;
 }
+
+.glyphicon.selected {
+  color: deepskyblue;
```

Comment:
Color `deepskyblue` should be written in hexadecimal form as `#00bfff`<br>Color literals like `deepskyblue` should only be used in variable declarations; they should be referred to via variable everywhere else.